### PR TITLE
[FS] Return proper JSON from Voting API

### PR DESCRIPTION
**Pull request by quantumagi**
See https://stratisplatformuk.visualstudio.com/StratisBitcoinFullNode/_workitems/edit/3786.

### CODE REVIEW

File: src/Stratis.Bitcoin.Features.PoA/Voting/VotingDataModel.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using NBitcoin;
+
+namespace Stratis.Bitcoin.Features.PoA.Voting
+{
+    public class VotingDataModel
+    {
+        public string key { get; private set; }
```
Comment (fassadlr) [inline src/Stratis.Bitcoin.Features.PoA/Voting/VotingDataModel.cs]: @MithrilMan also fix this to be uppercase and put in `[JsonProperty("key")]`

File: src/Stratis.Bitcoin.Features.PoA/Voting/VotingDataModel.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        public string hash { get; private set; }
```
Comment (fassadlr) [inline src/Stratis.Bitcoin.Features.PoA/Voting/VotingDataModel.cs]: @MithrilMan this as well

File: src/Stratis.Bitcoin.Features.PoA/Voting/VotingDataModel.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+        public VotingDataModel(VotingData votingData)
```
Comment (fassadlr) [inline src/Stratis.Bitcoin.Features.PoA/Voting/VotingDataModel.cs]: @MithrilMan  Also need to a private parameterless constructor and add [JsonConstructor] on it.

**Comment by MithrilMan** [comment]
fixed per review

**Pull request merged**